CHANGELOG.md

```
@@ -1,29 +1,53 @@
+## [4.2.0] - 07/07/2025
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)